docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 55 - 62
+```
+
+* Line 55 checks if the `prodNum` (i.e. product number) entered doesn't match any product or if the `prodAmt` entered is zero (0).
+
+* Line 56 `{!rsh} commit`s if Line 55 is true.
+
+* Line 57 calls the `reportCancellation` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 58 calls the `reportExit` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 59 terminates the transaction by calling the `{!rsh} exit` method.
+
+* Line 61 `{!rsh} commit`s if Line 55 is false.
+
+
+Next, type the following code:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 64 - 73
+```
+
+* Line 64 starts another `seller`'s only block.
+
+* Line 65 calculates the total price to be paid by the `buyer` depending on the `product` selected. This is calculated by multiplying the `price` of the `product` by the quantity that the buyer wants.
+
+* Line 67 publishes the total price.
+
+* Line 68 ends this local step.
+
+* Line 70 starts another `buyer`'s only block.
+
+* Line 71 calls the `confirmPurchase` `{!rsh} function` and passes the `total` from line 65 to it.
+This obtains the `buyer`'s consent and stores it as `willBuy`.
+
+* Line 73 publishes `willBuy`
+
+
+Now enter the code below:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 75 - 82
+```
+
+
+* Line 75 checks if the `willBuy` is false.
+
+* Line 76 `{!rsh} commit`s if Line 75 is true.
+
+* Line 77 calls the `reportCancellation` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 78 calls the `reportExit` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 79 terminates the transaction by calling the `{!rsh} exit` method.
+
+* Line 81 `{!rsh} commit`s if Line 75 is true.
+
+
+Finally, the following code handles payment and delivery if all goes well:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 84 - 90
+```
+
+* Line 84 has the `buyer` pay the `total` into the contract.
+
+* Line 85 calls the `reportPayment` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 86 transfers the money paid by the `buyer` to the `seller`.
+
+* line 87  calls the `reportTransfer` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* line 88  calls the `reportFulfillment` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* line 89  calls the `statusReport` `{!rsh} function` on each of the `{!rsh} Participant`.
+
+* Line 90 `{!rsh} commit` to the program.
+
+And That concludes the application.
+
+You have successfully completed an interactive command-line DApp.
+Congratulations on this new feat! 
+
+
+#### {#tut-mkt-Interaction-advance-test} Testing the application
+
+Open two terminals. In the first one, 
+run `{!cmd} ./reach run index seller` 
+and in the second terminal, 
+run `{!cmd} ./reach run index buyer`
+
+If the transaction is canceled, the output received should look like this:
```

Comment:
cancelled

both spellings are OK, but in the code, you have cancelled so we need to be consistent